### PR TITLE
Updates the Subatomic configuration (with encoders)

### PR DESCRIPTION
**Pull request by jackhumbert**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

File: keyboards/subatomic/keymaps/default/keymap.c [hunk 1]
```diff
@@ -1,17 +1,20 @@
-#include QMK_KEYBOARD_H
+#include "subatomic.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+#ifdef AUDIO_ENABLE
+  #include "audio.h"
+#endif
```
Comment (noroadsleft) [inline keyboards/subatomic/keymaps/default/keymap.c]: This delta should be reverted to

```c
#include QMK_KEYBOARD_H
```

File: keyboards/subatomic/keymaps/default/keymap.c [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
-    _LOWER,
-    _RAISE,
-    _ADJUST,
-};
+#define _QWERTY 0
+#define _COLEMAK 1
+#define _DVORAK 2
+#define _LOWER 3
+#define _RAISE 4
+#define _ADJUST 16
```
Comment (noroadsleft) [inline keyboards/subatomic/keymaps/default/keymap.c]: This delta should be reverted to

```c
enum layer_names {
    _QWERTY,
    _COLEMAK,
    _DVORAK,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```

File: keyboards/subatomic/keymaps/default/keymap.c [hunk 3]
```diff
@@ -22,133 +25,138 @@ enum subatomic_keycodes {
   BACKLIT
 };
 
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
+
```
Comment (noroadsleft) [inline keyboards/subatomic/keymaps/default/keymap.c]: These lines should be deleted.

**Comment by drashna** [comment]
@jackhumbert looks like this PR needs some TLC

**Comment by tzarc** [comment]
Swapped to develop, removed the label preventing stalebot from doing its thing. Needs #7029 apparently.

**Pull request closed without merging**